web/styles/reactions.css

```diff
@@ -267,6 +267,20 @@
                 height: 1.6667em; /* 25px at 15px/em */
                 width: 1.6667em; /* 25px at 15px/em */
             }
+
+            .animated-emoji {
+                display: none;
+            }
+
+            &:hover {
+                .still-emoji {
+                    display: none;
+                }
+
+                .animated-emoji {
+                    display: inline;
+                }
+            }
```

Comment:
Would it make sense for these properties to just have one copy inside `.emoji`, rather than one in each component that displays them?

Also, are there settings where emoji are in `flex` context and thus `inline` would be wrong?  I could imagine just having a `.animated-emoji` with `not(:hover)` rather than the explicit `display: inline`.